graphkb/match.py

```diff
@@ -269,7 +269,7 @@ def positions_overlap(
     if pos is None:
         return True
 
-    start = range_start.get('pos', None)
+    start = range_start.get('pos', None) if range_start else None
```

Comment:
Hi @creisle  - added a test match.match_positional_variant(conn, "ERBB2:p.G776_V777insYVMA")

If this change is inappropriate here, the fix probably belongs around line 320 in compare_positional_variants 'break2Start'

Error occurs when variant:
{'break1Repr': 'p.G776',
 'break1Start': {'@class': 'ProteinPosition', 'pos': 776, 'refAA': 'G'},
 'break2Repr': 'p.V777',
 'break2Start': {'@class': 'ProteinPosition', 'pos': 777, 'refAA': 'V'},
 'reference1': 'ERBB2',
 'type': 'insertion',
 'untemplatedSeq': 'YVMA',
 'untemplatedSeqSize': 4}

is compared with overlap to
{'@class': 'PositionalVariant',
 '@rid': '#157:5403',
 'break1Repr': 'p.G776',
 'break1Start': {'@class': 'ProteinPosition', 'pos': 776, 'refAA': 'G'},
 'break2Repr': 'p.?776',
 'break2Start': None,
 'createdAt': 1621292111312,
 'createdBy': '#30:0',
 'displayName': 'ERBB2:p.G776delGinsVV',
 'germline': False,
 'history': '#159:5352',
 'refSeq': 'G',
 'reference1': '#127:91527',
 'type': '#145:17',
 'untemplatedSeq': 'VV',
 'updatedAt': 1621292262454,
 'updatedBy': '#30:0',
 'uuid': '05ff933f-8d85-4102-9d0b-7b42ff804741'}

Reply:
another thing that's weird here is the second one has a "break2Repr" but no break2.

Reply:
ya i think this should be in the other function. Made separate PR https://github.com/bcgsc/pori_graphkb_python/pull/43 since its such a small thing was easier than describing. Can we close this one and use that one instead?